hicommonwealth/commonwealth
# Migration for arbitrary social links in Community table

## Link to Issue
Closes: #5815

## Description of Changes
- Changes how we handle social links in Communities table. Stores as an array so we can arbitrarily add new social links.
- Adds field for Community namespaces, for later use

## Test Plan
- Create a community, fill in social links.
- Go to community discussions page, click on social link icons in bottom left corner, make sure it navigates correctly to the  correct url.
